### ISSUE TITLE
Endorsements on haskell.org homepage

### ISSUE THREAD

**Issue by kosmikus** [haskell-infra/www.haskell.org#289]
What is the intended role of the "Endorsements" section on the homepage?

I was wondering whether it would be good to have Well-Typed add an endorsement. I originally thought that the answer is "no", because it would be better to have endorsements by "end users" of Haskell, i.e., companies that actually develop products in Haskell on their own, as opposed to consultancies.

But then I noticed that there are consultancies (e.g. Serokell, Stack Builders) in the list. So is there an official policy / guideline about this? Would a PR adding an endorsement from Well-Typed be accepted?

**Comment by tomjaguarpaw** [comment]
I don't think we have a policy, per se. Currently all Haskell-using companies are welcome. We would definitely accept a PR adding an endorsement from Well-Typed!

**Comment by KareddyIndirapriyadarshini** [comment]
Hi! I am just starting to contribute to the platform and would like this to be a learning experience. Since I am not directly associated with Well-Typed, can I help with adding this endorsement to the website? Please let me know if that is okay!

**Comment by tomjaguarpaw** [comment]
@kosmikus, would you like to move forward with this?

**Comment by tomjaguarpaw** [comment]
And endorsement from Well Typed would be welcome so please open a PR to add one at any point in the future if you wish.
